### PR TITLE
a few updates

### PR DESCRIPTION
**Pull request by aserhatdemir**
collision detection problem fixed.
seperated campaigns
desert background

### CODE REVIEW

**Review by umuterturk — CHANGES_REQUESTED** [review]
I couldn't run the game starting from Menu scene
`Scene 'SelectCampaign' couldn't be loaded because it has not been added to the build settings or the AssetBundle has not been loaded.
To add a scene to the build settings use the menu File->Build Settings...
UnityEngine.SceneManagement.SceneManager:LoadScene(String)
<FadeOut>d__4:MoveNext() (at Assets/Scripts/SceneFader.cs:47)
UnityEngine.SetupCoroutine:InvokeMoveNext(IEnumerator, IntPtr)
`

File: Assets/CampaignSelect.cs [hunk 1]
```diff
@@ -4,9 +4,14 @@ public class CampaignSelect : MonoBehaviour
 {
     public SceneFader sceneFader;
     
-    public void PlayCampaing1()
+    public void PlayCampaign1()
```
Comment (umuterturk) [inline Assets/CampaignSelect.cs:7]: this naming is just bad

**Comment by umuterturk** [comment]
oil tower aspect ratio is no good.

**Comment by aserhatdemir** [comment]
Nice catch. Thanks for the feedback. i will push with flame and explosion effects included.

**Pull request merged**